docs/errors/E0717.md

```diff
@@ -0,0 +1,19 @@
+# E0717: import alias cannot use 'import type'
```

Comment:
Minor: This syntax should be called a **namespace alias**, not an **import alias**. quick-lint-js currently confuses the two terms, so I don't blame you for writing "import alias" here. There's a separate task to fix the wording throughout quick-lint-js (#1138).

Reply:
changed "import" to "namespace" and corrected the diagnostic name